example/lib/interactive_features_example.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:mapbox_maps_flutter/mapbox_maps_flutter.dart';
+import 'example.dart';
+
+class InteractiveFeaturesExample extends StatefulWidget implements Example {
+  @override
+  final Widget leading = const Icon(Icons.map);
+  @override
+  final String title = 'Interactive Features';
+  @override
+  final String? subtitle = 'Tap a Buildings to highlight it or a POI to hide it';
+
+  @override
+  State<StatefulWidget> createState() => InteractiveFeaturesState();
+}
+
+class InteractiveFeaturesState extends State<InteractiveFeaturesExample> {
+  InteractiveFeaturesState();
+  MapboxMap? mapboxMap;
+
+  _onMapCreated(MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+    mapboxMap.style;
+
+    /// Define interactions for 3D Buildings
```

Comment:
Note that these building interactions will not work with the STANDARD style. Use STANDARD_EXPERIMENTAL instead to test.